test/test_postman.py

```diff
@@ -314,9 +315,12 @@ def test_postman():
 
     # Test initializing the l2 example contract
     l2_contract_address = _init_l2_contract(l1l2_example_contract.address)
-
     _l1_l2_message_exchange(web3, l1l2_example_contract, l2_contract_address)
 
+    # Check if all blocks were generated
+    latest_block = get_block(parse=True)
+    assert latest_block["block_number"] == 4
```

Comment:
Do you want to check what is inside the block and compare block data with some expected data?

Do you want to modify the l1l2 contract to emit events and also check events?

Reply:
If you remember the way you tested this during development, with those few CLI calls/checks that you did to make sure everything was populated propertly. Why not do that here? Not necessarily match whole responses, but at least asserting that the L1_HANDLER tx was executed and stored.

The current checking if the latest block number is 4 is not enough and we might soon forget why it is 4.

Reply:
Ok so I'll change that to check if tx is L1_HANDLER.

And what about events? Do we also want to change that?

Reply:
Done https://github.com/Shard-Labs/starknet-devnet/pull/319/commits/1e6bcbe11e8c07833b8de41629f47fc838128ec6

Reply:
Actually the event can be made more meaningful. It could be called something like `balance_increased` and contain the new value. And then the assertion could be made on the event content.

Reply:
Changed after our call https://github.com/Shard-Labs/starknet-devnet/pull/319/commits/432c81cdb3570aa4884e462888cc49a29af9819b